src/native/selection_list.rs

```diff
@@ -23,7 +27,7 @@ where
     /// Label Font
     font: Renderer::Font,
     /// Style for Looks
-    style: selection_list::Style,
+    style: Style,
```

Comment:
shouldn't this be `style_sheet: Box<dyn StyleSheet + 'a>,`?

Reply:
sadly Selection list never used the stylesheet and only the style directly. IDR why we decided to do it this way but it was due to some reason. maybe a lifetime issue.

Reply:
okay, than keep it that way :+1: